docs/admin_api/rooms.md

```diff
@@ -1,13 +1,9 @@
 # Contents
```

Comment:
since there's an auto-generated ToC at https://matrix-org.github.io/synapse/develop/admin_api/rooms.html, I'd be inclined to just get rid of this.

Reply:
I think so, too. But there is a discussion in #10127 to resolve it with CSS.